### DISCUSSION TITLE
コマンド入力例のCopyにプロンプトは不要だと思います

### DISCUSSION THREAD

**Discussion by iMasaruOki** [increments/qiita-discussions#538] [要望・アイデア・不具合]
Qiitaさんの記事ページで、\`\`\`で囲まれてるコードブロックにマウスをかざすと右上にマークが出て、クリックすれば内容をペーストバッファにコピーできます。便利ですね。

で、\`\`\`ConsoleのときCopyでプロンプト部分を消してくれるか気になったのでテストしてみました。

```console
$ sudo apt update
$ sudo apt install -y foo bar
```

記事にすると色でプロンプト自体は認識してるっぽいですが、はたして。

ペーストしてみると

```
$ sudo apt update
$ sudo apt install -y foo bar
```

残念。こうだったかな、と思って記事ではプロンプトを省いて例を書いてましたが、記憶違いではなかったようです。`$ `を省いてくれると嬉しいのだけど! (シェルにコピペしてEnterで実行できる)

Consoleのときだけとか、rootだと`#`になるとかホスト名とか面倒かと思いますが(あるいは別キーワードで動作を明示してもいいです)、改善されると嬉しいな! という意見表明でした。
文体砕けててすみません。ご検討いただければ幸いです。

**Comment by wataru86** [comment]
@iMasaruOki

ご意見ありがとうございます🙏

コマンドのみコピーしたいという需要もあると思うのですが、プロンプトの種類が様々であったり、プロンプトも含めてコピーしたい場面もあるので、コピーの動作を変更するのは難しいと考えています。
ご要望が多ければ改めて検討したいと思います🙇
